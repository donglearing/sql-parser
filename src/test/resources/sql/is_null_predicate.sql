select a from T where a is null and b > 2;