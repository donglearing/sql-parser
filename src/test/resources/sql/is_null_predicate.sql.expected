SELECT a FROM T WHERE a IS NULL AND b > 2